modules/ROOT/pages/upgrade-message-flows.adoc

```diff
@@ -10,7 +10,7 @@ Runtime templates related to Partner Manager are upgraded when:
 * Functional and performance defects are resolved in the template
 * Flow optimizations are implemented
 
-When new runtime templates are available, the *Manage deployments* button is enabled with these options:
+When new runtime templates are available, the *Manage deployments* button changes color and shows a tooltip to indicate a new version is available. To take advantage of the new features, click *Manage deployments* and select one of these options:
```

Comment:
There is no change in color. When new runtime templates are available, just a tool tip is shown.
Users can always click Manage deployments button regardless of new versions being available or not.
When they click they go to the next screen, where the first radio button gets enabled only when a new template version is available. The second radio button is enabled all the time.